lunes_cms/cms/migrations/0005_document_created_by_link.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.13 on 2022-07-05 11:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def link_created_by(apps):
+    Document = apps.get_model("created_by", "Document")
+    Group = apps.get_model("created_by", "Group")
```

Comment:
```suggestion
    Document = apps.get_model("cms", "Document")
    Group = apps.get_model("auth", "Group")
```
The first parameter of the `get_model` function is the app name